packages/react-button/package.json

```diff
@@ -14,26 +14,23 @@
   },
   "license": "MIT",
   "scripts": {
-    "build": "just-scripts build",
-    "bundle": "just-scripts bundle",
-    "clean": "just-scripts clean",
-    "code-style": "just-scripts code-style",
-    "just": "just-scripts",
-    "lint": "just-scripts lint",
-    "start": "just-scripts dev:storybook",
-    "start:legacy": "just-scripts dev",
-    "start-test": "just-scripts jest-watch",
-    "test": "just-scripts test",
-    "update-api": "just-scripts update-api",
-    "update-snapshots": "just-scripts jest -u"
+    "build": "echo 'TODO: rollup & tsc --declaration && api-extractor'",
+    "clean": "rm -rf dist lib-{amd,commonjs,es-2015} .cache",
+    "start": "yarn serve",
+    "serve": "start-storybook --ci -c ./.storybook -p 6006",
+    "test": "jest",
+    "lint": "eslint --ext=.ts,.tsx src/",
+    "format": "prettier **/*.{ts,tsx,md,html,scss} --write",
```

Comment:
One concern I have with duplicating the configuration for each command in every package.json is that it would have a negative impact on maintainability (if the options need to be changed for some reason, as far as I can tell you'd have to rely on find/replace and hope it caught everywhere). What has been your experience with that in other projects?

Reply:
Got it, I wasn't clear on whether the manual configured commands in each package.json were intended as part of the proposal, or just a way to make things work right now (sounds like it's the latter, so I'm not too worried about it for now).

Reply:
yup your latter statement. thx